vendor/assets/stylesheets/rolodex/components/_forms.sass

```diff
@@ -31,7 +31,8 @@ input[type="search"]
   box-sizing: border-box
 
 input[type="search"]::-webkit-search-cancel-button,
-input[type="search"]::-webkit-search-decoration
+input[type="search"]::-webkit-search-decoration,
+input[type="submit"]
   -webkit-appearance: none
```

Comment:
Since we're using it in almost all of our front end app, how will `autoprefixer` handle us using `-webkit-appearance` instead of `appearance`?